### PR TITLE
calculate SE variables for IEA

### PR DESCRIPTION
**Pull request by fbenke-pik**

### CODE REVIEW

File: R/calcFE.R [hunk 1]
```diff
@@ -59,6 +59,24 @@ calcFE <- function(source = "IEA", scenario_proj = "SSP2") {
     x <- mbind(x, setNames(x[, , "SE|Heat|Biomass|CHP (EJ/yr)"] +
       x[, , "SE|Heat|Biomass|HP (EJ/yr)"], "SE|Heat|Biomass (EJ/yr)"))
 
+
+    # add secondary energy totals
+    x <- mbind(x, setNames(x[, , "SE|Gases|Biomass (EJ/yr)"]
+                           + x[, , "SE|Gases|Fossil|Coal (EJ/yr)"]
+                           + x[, , "SE|Gases|Fossil|Natural Gas (EJ/yr)"], "SE|Gases (EJ/yr)"))
+
+    x <- mbind(x, setNames(x[, , "SE|Liquids|Bio Oil (EJ/yr)"]
```
Comment (Renato-Rodrigues) [inline R/calcFE.R:68]: why not use the variable `SE|Liquids|Biomass`? Is `SE|Liquids|Bio Oil` equal to `SE|Liquids|Biomass`?

**Comment by Renato-Rodrigues** [comment]
could you also check/add these additional variables together with this PR?

`SE|Gases|Fossil = SE|Gases|Fossil|Coal + SE|Gases|Fossil|Natural Gas`
`SE|Heat|Fossil = SE|Heat|Coal + SE|Heat|Gas`
`SE|Liquids|Fossil = SE|Liquids|Fossil|Gas + SE|Liquids|Fossil|Oil`
`SE|Solids|Fossil = SE|Solids|Coal` (<- I am not sure about this one just from memory if it also needs to include some waste term)

File: R/calcFE.R (hunk 1 above)
Comment (fbenke-pik) [reply R/calcFE.R:68]: `getNames(x[, , "SE|Liquids", pmatch=T])`  yields
`"SE|Liquids|Bio Oil (EJ/yr)"    "SE|Liquids|Fossil|Gas (EJ/yr)" "SE|Liquids|Fossil|Oil (EJ/yr)"` 

`SE|Liquids|Biomass`  seems to be not available in this context  (uses `structuremappingIO_reporting.csv`)

Comment (robertpietzcker) [reply R/calcFE.R:68]: This is actually what got me started again with the structuremappings....
https://github.com/remindmodel/development_issues/issues/83#issuecomment-1801368318

I don't think this is an output that is produced from structuremappingIO_reporting, and before changing that I want to make sure it builds up from the right structuremapping...

**Comment by robertpietzcker** [comment]
> could you also check/add these additional variables together with this PR?
> 
> `SE|Gases|Fossil = SE|Gases|Fossil|Coal + SE|Gases|Fossil|Natural Gas` `SE|Heat|Fossil = SE|Heat|Coal + SE|Heat|Gas` `SE|Liquids|Fossil = SE|Liquids|Fossil|Gas + SE|Liquids|Fossil|Oil` `SE|Solids|Fossil = SE|Solids|Coal` (<- I am not sure about this one just from memory if it also needs to include some waste term) `SE|Electricity|Fossil = SE|Electricity|Coal + SE|Electricity|Gas + SE|Electricity|Oil` or `SE|Electricity|Fossil|w/ CC = SE|Electricity|Fossil|w/ CC + SE|Electricity|Fossil|w/o CC`

actually, I think it has to be
`SE|Heat|Fossil = SE|Heat|Coal + SE|Heat|Gas + SE|Heat|Oil` 
`SE|Liquids|Fossil = SE|Liquids|Fossil|Gas + SE|Liquids|Fossil|Oil + SE|Liquids|Coal`

**Comment by fbenke-pik** [comment]
Just agreed with Robert that we will first look into unifying / fixing the structure mappings (see https://github.com/remindmodel/development_issues/issues/83#issuecomment-1801368318) before moving forward with this.

**Pull request closed without merging**